packages/playwright-core/src/server/android/android.ts

```diff
@@ -264,9 +264,13 @@ export class AndroidDevice extends SdkObject {
     debug('pw:android')('Force-stopping', pkg);
     await this._backend.runCommand(`shell:am force-stop ${pkg}`);
     const socketName = isUnderTest() ? 'webview_devtools_remote_playwright_test' : ('playwright-' + createGuid());
-    const commandLine = this._defaultArgs(options, socketName).join(' ');
+    const commandLine = this._defaultArgs(options, socketName).map(arg => {
```

Comment:
see [here](https://source.chromium.org/chromium/chromium/src/+/main:base/android/java/src/org/chromium/base/CommandLine.java;l=175;drc=38321ee39cd73ac2d9d4400c56b90613dee5fe29;bpv=1;bpt=1) for the upstream command-line parser